### ISSUE TITLE
BC break: expressions and macros inside property-block definition are not expanded

### ISSUE THREAD

**Issue by peci1** [ros/xacro#108]
```
<root version="1.0" xmlns:xacro="http://www.ros.org/wiki/xacro">
        <xacro:macro name="test">
                ahoj
        </xacro:macro>
        <xacro:property name="val" value="2" />
        <xacro:property name="prop">
                <outer attr="${val}">
                testy
                <xacro:test />
                </outer>
        </xacro:property>
        <xacro:macro name="final">
                <xacro:insert_block name="prop" />
        </xacro:macro>
        <xacro:final />
</root>
```

This snippet generates the following in Indigo:

```
<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from test.xacro
        | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED
        | -->
<!-- =================================================================================== -->
<root version="1.0" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <outer attr="2">

                testy


                ahoj

  </outer>
</root>
```

And the following in Jade (regardless of --inorder):

```
<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from test.xacro
        | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED
        | -->
<!-- =================================================================================== -->
<root version="1.0" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <outer attr="${val}">

                testy

    <xacro:test/>
  </outer>
</root>
```

**Comment by rhaschke** [comment]
@codebot: I will have a look.
Best, Robert

**Comment by rhaschke** [comment]
Fixed with #109.
